applications/nrf5340_audio/src/bluetooth/le_audio_bis_gateway.c

```diff
@@ -391,13 +391,27 @@ int le_audio_play_pause(void)
 	return ret;
 }
 
-int le_audio_send(uint8_t const *const data, size_t size)
+int le_audio_send(struct encoded_audio enc_audio)
 {
 	int ret;
 	static bool wrn_printed[CONFIG_BT_AUDIO_BROADCAST_SRC_STREAM_COUNT];
 	struct net_buf *buf;
 	size_t num_streams = ARRAY_SIZE(audio_streams);
-	size_t data_size = size / num_streams;
+	size_t data_size_pr_stream;
+
+	if (enc_audio.num_ch == 1) {
+		data_size_pr_stream = enc_audio.size;
+	} else if (enc_audio.num_ch == num_streams) {
+		data_size_pr_stream = enc_audio.size / num_streams;
+	} else {
+		LOG_ERR("Number of encoded channels must be 1 or equal to the number of streams");
```

Comment:
This will also change when we support to send two channels per stream, for headsets for instance. Then we might have num_ch== 2 and num_streams == 1

Reply:
Come to think about it I don't think we should return an error if num_ch is bigger than num_streams. Yes, there is some power lost in terms of encoding time, but we could have BIS and CIS concurrently where the BIS only does mono whereas the CIS does stereo.